src/latex2sympy/units/unit_aliases/__init__.py

```diff
@@ -0,0 +1,17 @@
+import os.path
```

Comment:
@builtbywill These changes all look wonderful. One additional thought I had this morning - we need to have a way to delete this file if the code changes. I'm thinking maybe we want something like the library's version number included in the filename? I can make that change?

Reply:
@jkdufair I had been thinking if you need to make changes to `unit_aliases.py` you just need to run that file to re-generate the pickle and the json, which you would always be doing to check that the aliases exist in the json

I could leave a note at the top of the file?

Reply:
My original thought, I'm sure you saw, was to have the pkl file auto-generate. I still think that's a good idea vs. checking it into git. It'll only need to happen once per release (per instance) and then we can regenerate it when we have new releases.

Though maybe you put it in git because the filesystem for Azure functions is ephemeral? I get the impression they are. So what about generating it as a step in the cmake build file? We have to do that with every release, yeah?

Reply:
Or, more simply, right in the `compile.sh` script?

Reply:
yeah i don't think we want to (or possibly even can) be writing files to the file system while hosted in the azure function, and it would have to happen on first run of each instance, which happens all the time

no, the compile script only needs to be run if you actually make changes to the parser grammar or the CPP code

i would see making changes to `unit_aliases.py` as a similar but separate workflow, where when you make changes to it, you have run it to "compile" it. if you are actively developing that file, you _need_ to run it to export the JSON anyways to check that. or if not that, you would probably be adding unit tests, which would then only succeed once you compile the pickle file, since the tests will be running against that.

Reply:
OK here's an idea. A unit test that generates a hash of `unit_aliases.py`. If it doesn't match what's in the current unit test, it tells you you need to regenerate the pkl and json? Then we won't ever have to worry about forgetting to do that. We could even do the same for the parser grammar or CPP code. A small gift to future us.

Reply:
I'm gonna try that out here for a minute

Reply:
@builtbywill Ok added